config/routes.rb

```diff
@@ -1,4 +1,11 @@
 Rails.application.routes.draw do
+  get 'user_sessions/new'
+
+  get 'user_sessions/create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.